app/src/main/java/com/example/gfood/data/model/Category.kt

```diff
@@ -1,11 +1,13 @@
 package com.example.gfood.data.model
 
+import java.io.Serializable
+
 data class Category(
     val id: String?,
     val name: String?,
     val image: String?,
     val description: String?
-)
+) : Serializable
```

Comment:
Serializable: thằng này dùng làm gì vậy e?

Reply:
Dùng thằng này để gửi object sang activity khác bằng intent á anh

Reply:
cơ chế hoạt động của nó là gì?
Không dùng nó gửi qua intent được không e?

Reply:
Dạ.Nó là quá trình chuyển các cấu trúc dữ liệu và các đối tượng thành một định dạng có thể lưu trữ được (vào file, in-memory buffer, hoặc truyền qua network), sau đó có thể phục hồi lại các cấu trúc dữ liệu và đối tượng như ban đầu, trên cùng hoặc khác môi trường. 
Ngoài nó ra thì chúng ta có thể sử dụng Parcelable để truyền dữ liệu. Nhưng theo lí thuyết thì Parcelable vẫn tối ưu hơn ạ

Reply:
Dạ nó là quá trình chuyển các cấu trúc dữ liệu và các đối tượng thành một định dạng có thể lưu trữ được (vào file, in-memory buffer, hoặc truyền qua network), sau đó có thể phục hồi lại các cấu trúc dữ liệu và đối tượng như ban đầu, trên cùng hoặc khác môi trường.
Nếu không dùng nó thì phải dùng Parcelable thì mới gửi object qua intent được ạ